sdcm/cluster.py

```diff
@@ -3662,11 +3672,22 @@ def update_scylla_packages(node, _queue):
             node.remoter.run('tar -xvf /tmp/scylla/*.tar.gz -C /tmp/scylla/', ignore_status=True, verbose=True)
 
             # replace the packages
-            logging.info("Installing rpms")
-            node.remoter.run('yum list installed | grep scylla')
-            node.remoter.sudo('rpm -URvh --replacepkgs --replacefiles /tmp/scylla/*.rpm',
-                              ignore_status=False, verbose=True)
-            node.remoter.run('yum list installed | grep scylla')
+            self.log.debug(f'Node distro is {node.distro}, will check the package suffix received')
+            if node.distro.is_rhel_like:
+                check_package_suites_distro(node, 'rpm')
+                self.log.info('Installed RPMs before replacing with new RPM files')
+                node.remoter.run('yum list installed | grep scylla', verbose=True)
+                self.log.debug('Will replace the RPM files with the ones provided as a parameter to the test')
+                node.remoter.sudo('rpm -URvh --replacefiles /tmp/scylla/*.rpm', ignore_status=False, verbose=True)
+                self.log.info('Installed RPMs after replacing with new RPM files')
+                node.remoter.run('yum list installed | grep scylla', verbose=True)
+            elif node.distro.is_ubuntu:
+                check_package_suites_distro(node, 'deb')
+                self.log.info('Installed .deb packages before replacing with new .DEB files')
+                node.remoter.run('apt list --installed | grep scylla', verbose=True)
+                node.remoter.sudo('apt install /tmp/scylla/scylla*', ignore_status=False, verbose=True)
+                self.log.info('Installed .deb packages after replacing with new .DEB files')
+                node.remoter.run('apt list --installed | grep scylla', verbose=True)
```

Comment:
Was this tested?
According to @ShlomiBalalis it doesn't work.

Also,  please find if there  is a flag that can run without dependencies.

Reply:
[this test](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fabio/job/scyllakillmonkey-reduce-severity-c-s-error/16/) used this very last code, and finished green.
i'm checking the print of this command in `sct.log`, but it was more for debug purpose.

and another thing, this code changed since @ShlomiBalalis took the code from my branch

Reply:
@roydahan , yes, it worked, output in last test:
```
Running command "apt list --installed | grep scylla"...

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Preparing to unpack .../6-scylla-tools-core_4.6.dev-0.20210629.edd54a946-1_all.deb ...
Unpacking scylla-tools-core (4.6.dev-0.20210629.edd54a946-1) over (4.6.dev-0.20210628.c0c1e2601-1) ...
scylla-conf/now 4.6.dev-0.20210629.edd54a946-1 amd64 [installed,local]
scylla-jmx/now 4.6.dev-0.20210629.edd54a946-1 all [installed,local]
scylla-kernel-conf/now 4.6.dev-0.20210629.edd54a946-1 amd64 [installed,local]
scylla-machine-image/stable,stable,now 4.6.dev-20210629.49a8d0ec75-1 all [installed]
scylla-node-exporter/now 4.6.dev-0.20210629.edd54a946-1 amd64 [installed,local]
scylla-python3/now 4.6.dev-0.20210629.edd54a946-1 amd64 [installed,local]
scylla-server-dbg/now 4.6.dev-0.20210629.edd54a946-1 amd64 [installed,local]
scylla-server/now 4.6.dev-0.20210629.edd54a946-1 amd64 [installed,local]
scylla-tools-core/now 4.6.dev-0.20210629.edd54a946-1 all [installed,local]
scylla-tools/now 4.6.dev-0.20210629.edd54a946-1 all [installed,local]
scylla/now 4.6.dev-0.20210629.edd54a946-1 amd64 [installed,local]
STDERR: 
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Command "apt list --installed | grep scylla" finished with status 0
```

Reply:
> Also, please find if there is a flag that can run without dependencies.

@roydahan , what does it mean?

Reply:
1. The install doesn't work, not the apt list.
2. In case there are dependenices that aren't  met, there may be a flag to override it.
3. At the moment, Shlomo is using a different command that works "dpkg -i ..."

Reply:
the broken dependencies could be added, but it could also break Scylla...
and the install command does work (when the full dependencies are met), as these `20210629.edd54a946` are the files provided in the test

Reply:
@roydahan @ShlomiBalalis , i added to the command `--ignore-missing` and it should do it

Reply:
@roydahan , with Shlomo's .deb files it still doesn't work, and anyway, running as you said, `dpkg -i  <files>` finishes with failure too (although it does install the ones in the temp folder), meaning that we will need to run with `ignore_status=True` what i'm not sure is the best practice for this case, and i couldn't find other command to do the job without failing.
WDYT?

@bentsi @enaydanov @dkropachev , your opinions here are very welcome to this situation